### PR TITLE
Rate limit v2

### PR DESCRIPTION
**Pull request by shulkaolka**
#129

### CODE REVIEW

File: lib/helpers.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+      rateLimitDelay = delay * 1000;
+    }
+    logger.info('rateLimitDelay is:', rateLimitDelay);
+  }
+  return rateLimitDelay;
+}
+
+async function rateLimit(logger, delay) {
+  if (delay) {
+    logger.info('Delay Between Calls is set to:', delay);
```
Comment (kirill-levitskiy) [inline lib/helpers.js:65]: I'd recommend to replace logger.info with logger.debug here, to reduce amount of logs

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
```
[the remark this answers is not in the document]
Comment (shulkaolka) [reply package.json:1]: updated

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,6 @@
+## 1.2.8 (July 6, 2020)
+* Add configuration fields: `Delay` and `Call Count` for rate limit
+
```
[the remark this answers is not in the document]
Comment (shulkaolka) [reply CHANGELOG.md:4]: done

File: README.md [hunk 4]
```diff
@@ -45,6 +45,10 @@ The example below shows the development team creation using the REST API compone
     - 504: Gateway Timeout
     - DNS lookup timeout
 4. ``Follow redirect mode`` - If you want disable Follow Redirect functionality, you can use option ``Follow redirect mode``.By default ``Follow redirect mode`` option has value ``Follow redirects``.
+5. ``Delay`` - If you want to slow down requests to your API you can set delay value (in seconds) and the component will delay calling the next request after the previous request.
+Time for the delay is calculated as `Delay`/ `Call Count` and shouldn't be more than 1140 seconds (19 minutes due to platform limitation). 
+The `Call Count` value by default is 1. If you want to use another value, please set the `Call Count` field. 
+6. ``Call Count`` - the field should be used only in pair with `Delay`, default to 1.
```
[the remark this answers is not in the document]
Comment (shulkaolka) [reply README.md:51]: 50 line is part of `Delay` description, 51 line is description of separate field

File: lib/helpers.js (hunk 1 above)
Comment (shulkaolka) [reply lib/helpers.js]: I replaced all other level to debug, but left info for this line because we can see at log why exactly we are waiting for.

**Pull request merged**